cross-project-tests/tools/llvm-objdump/ARM/plt.c

```diff
@@ -0,0 +1,243 @@
+// Test PLT section parsing on arm.
+
+// RUN: %clang --target=armv6a-none-linux-gnueabi -fuse-ld=lld \
+// RUN:   -nostdlib -nostdinc -shared %s -o %t1
+// RUN: llvm-objdump --no-show-raw-insn --no-print-imm-hex \
```

Comment:
The tet tests v6a/v8m & LE/BE. Perhaps use a more meaningful executable name, say, `%v6` `%v6.be` than `%t1`.